lib/routes/ncku/phys.ts

```diff
@@ -0,0 +1,95 @@
+import type { Route } from '@/types';
+import { CheerioAPI, load } from 'cheerio';
+import ofetch from '@/utils/ofetch';
+
+const currentURL = (catagory: string) => `https://phys.ncku.edu.tw/news/${catagory === '_all' ? '' : catagory}/`;
```

Comment:
This causes [double slashes](https://github.com/DIYgod/RSSHub/actions/runs/11044369127/job/30680189066#step:13:20) `//` in the URL.